src/main/java/taxi/controller/car/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,31 @@
+package taxi.controller.car;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.service.CarService;
+
+public class GetMyCurrentCarsController extends HttpServlet {
+    private final CarService carService;
+
+    {
+        Injector injector = Injector.getInstance("taxi");
+        carService = (CarService) injector.getInstance(CarService.class);
+    }
```

Comment:
choose only one way how you initialize fields, in previous controller you don't use initialization block

Reply:
I would like to init in static block but another mentor on Q&A told me to replace it and i've forgotten about CurrentCarsController